assets/js/googlesitekit/modules/datastore/modules.js

```diff
@@ -821,6 +825,34 @@ const baseSelectors = {
 
 		return screenWidgetContext || null;
 	} ),
+
+	/**
+	 * Gets the module's list of features.
+	 *
+	 * Returns a list of features of this module.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param {Object} state Data store's state.
+	 * @param {string} slug  Module slug.
+	 * @return {(Array|undefined)} An array of features for the module; `undefined` if state is still loading.
+	 */
+	getModuleFeatures: createRegistrySelector( ( select ) => ( state, slug ) => {
```

Comment:
If we're going to add a selector for this, it should get a test as well in the respective `modules.test.js` 🙂

Reply:
Test added @aaemnnosttv